styles.css

```diff
@@ -1,20 +1,199 @@
 * {
+    -webkit-box-sizing: border-box;
     box-sizing: border-box;
+    font-family: 'Catamaran', sans-serif;
 }
 
 body {
     font-size: 62.5%;
+    background-image: url("./img/34216.jpg");
+    background-attachment: fixed;
+    /*background-color: rgba(20, 20, 255, 1);*/
+}
+
+h1 {
+    font-size: 3rem;
+    margin: 0;
+    text-align: center;
+}
+
+p {
+    font-size: 2rem;
+    text-align: center;
 }
 
 img {
-    width: 200px;
+    height: 90%;
+}
+
+.container {
+    background-color: rgba(255, 255, 255, 0.6);
+    padding: 20px;
+}
+
+.guitarsContainer {
+    border: 1px gray solid;
+    background-color: rgba(240, 240, 255, 0.8);
+    width: 90%;
+    margin: 0 auto;
+    font-size: 1.6rem;
 }
 
 .row {
```

Comment:
None of these rules seem to do anything

Reply:
Agreed, the row class is always overridden